src/report.ts

```diff
@@ -20,9 +20,16 @@ async function pollReport(
 ): Promise<Result<AnalysisFailedResponse | ReportResult, GetAnalysisErrorCodes>> {
   // Return early if project name is not provided.
```

Comment:
nitties if nits. This comment is slightly off the place. Wonder if we need it here, since it's obvious what is done in the